### PR TITLE
gcc -> 12.2

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=gcc122  CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/gcc.rb [hunk 1]
```diff
@@ -137,14 +137,14 @@ def self.build
...
-             LDFLAGS: "-L#{CREW_LIB_PREFIX}/lib -Wl,-rpath=#{CREW_LIB_PREFIX}",
-        LIBRARY_PATH: CREW_LIB_PREFIX,
-                PATH: @path
+        NM: 'gcc-nm',
+        AR: 'gcc-ar',
+    RANLIB: 'gcc-ranlib',
+    CFLAGS: @cflags,
+  CXXFLAGS: @cxxflags,
+   LDFLAGS: "-L#{CREW_LIB_PREFIX}/lib -Wl,-rpath=#{CREW_LIB_PREFIX}",
+LIBRARY_PATH: CREW_LIB_PREFIX,
```
Comment (uberhacker) [inline packages/gcc.rb:146]: Indentation hiccup?

File: packages/gcc.rb [hunk 2]
```diff
@@ -178,8 +178,8 @@ def self.install
 
     make_env = {
       LIBRARY_PATH: CREW_LIB_PREFIX,
-      PATH: @path,
-      DESTDIR: CREW_DEST_DIR
+              PATH: @path,
+           DESTDIR: CREW_DEST_DIR
```
Comment (uberhacker) [inline packages/gcc.rb:182]: Too much indentation?

**Review by uberhacker — CHANGES_REQUESTED** [review]
Indentation improvements needed?

File: packages/gcc.rb (hunk 1 above)
Comment (satmandu) [reply packages/gcc.rb:146]: fixed

File: packages/gcc.rb (hunk 2 above)
Comment (satmandu) [reply packages/gcc.rb]: fixed

**Review by uberhacker — APPROVED** [review]

**Pull request merged**